ca/server.go

```diff
@@ -34,6 +37,8 @@ type Server struct {
 	store                       *store.MemoryStore
 	securityConfig              *SecurityConfig
 	joinTokens                  *api.JoinTokens
+	rootCA                      *api.RootCA
+	externalCAs                 []*api.ExternalCA
```

Comment:
Wondering if we should name these something different to avoid confusing them with actual `RootCA` and `ExternalCA` objects. Not sure what to suggest, though.